backend/ledgers.ts

```diff
@@ -20,19 +27,17 @@ function updateResults() {
     order by 1 desc`;
 
   postgres.sequelize
-    .query(query, { type: postgres.sequelize.QueryTypes.SELECT })
-    .then((results) => (cachedData = _.each(results, convertFields)));
+    .query(query, { type: QueryTypes.SELECT })
+    .then((results: Array<any>) => {
+      cachedData = _.each(results, convertFields);
+    });
 }
 
-function convertFields(ledger) {
-  // String to int fields
-  const fields = ["transaction_count", "operation_count"];
-  for (let field of fields) {
-    ledger[field] = parseInt(ledger[field]);
-  }
-
+function convertFields(ledger: any): Ledger {
```

Comment:
this one's because the `transaction_count` and `operation_count` values when pulled from postgres come back as strings. I thought it might be too confusing if there was a second Ledger type with `transaction_count` and `operation_count` as type `string` instead of `number`, so I left it as `any`. wdyt? 🤔

Reply:
Would it be OK to accept either `string` or a `number` for those props? That way we could use the same type in both places. Another option is to add an extra type that extends the main type. I personally like when everything is typed as much as possible, makes it easier to work with that code.

Reply:
an extra type sgtm 👍  agreed less uses of `any` is better
https://github.com/stellar/dashboard/pull/204/commits/eece86d470600eda80ae0f9d7c066fe0cfaf1f7a